src/init.js

```diff
@@ -0,0 +1,346 @@
+import { isObject } from 'lodash'
```

Comment:
It might help for clarity to call this file "createFirebaseInstance" or something along those lines.